app/demo-interactor/HostKNDemoRunner.hh

```diff
@@ -0,0 +1,64 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2020 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file HostKNDemoRunner.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include "physics/base/ParticleParams.hh"
+#include "physics/base/ParticleStatePointers.hh"
+#include "physics/em/KleinNishinaInteractorPointers.hh"
+#include "KNDemoIO.hh"
+#include "PhysicsArrayParams.hh"
+
+namespace demo_interactor_cpu
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Run interactions on the host CPU
```

Comment:
Minor comment: the [auto-brief](https://www.doxygen.nl/manual/docblocks.html) feature of doxygen requires an ending period: that's why I've started being pedantic about making complete sentences for the first bits 😬

Reply:
Didn't know that option.  I'll update them.